Src/ILGPU/Static/UnaryMathOperations.xml

```diff
@@ -0,0 +1,153 @@
+        <Implementation>{Const1} / {Value0}</Implementation>
+    </Operation>
+
+    <Operation Name="IsNaNF" IsPredicate="true">
+        <Summary>The is-not-a-number operation.</Summary>
+        <Flags>Floats</Flags>
+        <Call>IntrinsicMath.CPUOnly.IsNaN</Call>
+        <Implementation>{TypeName}.IsNaN({Value0})</Implementation>
+    </Operation>
+    <Operation Name="IsInfF" IsPredicate="true">
+        <Summary>The is-infinity operation.</Summary>
+        <Flags>Floats</Flags>
+        <Call>IntrinsicMath.CPUOnly.IsInfinity</Call>
+        <Implementation>{TypeName}.IsInfinity({Value0})</Implementation>
+    </Operation>
+
+    <Operation Name="SqrtF">
+        <Summary>Computes sqrt(value).</Summary>
+        <Flags>Floats</Flags>
+        <Call>IntrinsicMath.CPUOnly.Sqrt</Call>
+        <Implementation>{MathType}.Sqrt({Value0})</Implementation>
+    </Operation>
+    <Operation Name="RsqrtF">
+        <Summary>Computes 1/sqrt(value).</Summary>
+        <Flags>Floats</Flags>
+        <Call>IntrinsicMath.CPUOnly.Rsqrt</Call>
+        <Implementation>Rcp(Sqrt({Value0}))</Implementation>
+    </Operation>
+
+    <Operation Name="ASinF">
+        <Summary>Computes asin(x).</Summary>
+        <Flags>Floats</Flags>
+        <Call>IntrinsicMath.CPUOnly.Asin</Call>
+        <Implementation>{MathType}.Asin({Value0})</Implementation>
+    </Operation>
+    <Operation Name="SinF">
+        <Summary>Computes sin(x).</Summary>
+        <Flags>Floats</Flags>
+        <Call>IntrinsicMath.CPUOnly.Sin</Call>
+        <Implementation>{MathType}.Sin({Value0})</Implementation>
+    </Operation>
+    <Operation Name="SinHF">
+        <Summary>Computes sinh(x).</Summary>
+        <Flags>Floats</Flags>
+        <Call>IntrinsicMath.CPUOnly.Sinh</Call>
+        <Implementation>{MathType}.Sinh({Value0})</Implementation>
+    </Operation>
+
+    <Operation Name="ACosF">
+        <Summary>Computes acos(x).</Summary>
+        <Flags>Floats</Flags>
+        <Call>IntrinsicMath.CPUOnly.Acos</Call>
+        <Implementation>{MathType}.Acos({Value0})</Implementation>
+    </Operation>
+    <Operation Name="CosF">
+        <Summary>Computes cos(x).</Summary>
+        <Flags>Floats</Flags>
+        <Call>IntrinsicMath.CPUOnly.Cos</Call>
+        <Implementation>{MathType}.Cos({Value0})</Implementation>
+    </Operation>
+    <Operation Name="CosHF">
+        <Summary>Computes cosh(x).</Summary>
+        <Flags>Floats</Flags>
+        <Call>IntrinsicMath.CPUOnly.Cosh</Call>
+        <Implementation>{MathType}.Cosh({Value0})</Implementation>
+    </Operation>
+
+    <Operation Name="TanF">
+        <Summary>Computes tan(x).</Summary>
+        <Flags>Floats</Flags>
+        <Call>IntrinsicMath.CPUOnly.Tan</Call>
+        <Implementation>{MathType}.Tan({Value0})</Implementation>
+    </Operation>
+    <Operation Name="TanHF">
+        <Summary>Computes tanh(x).</Summary>
+        <Flags>Floats</Flags>
+        <Call>IntrinsicMath.CPUOnly.TanH</Call>
```

Comment:
ISSUE: The unit test in ILGPU.Algorithms is failing due to changing the case from `Tanh` to `TanH`. I think it is related to remapping intrinsics. Not sure why `ACos` and `ASin` are unaffected.

Reply:
I have tried to unify all names; however, it makes more sense to keep the old names, since the .Net `Math` class uses the same names.